src/Utils/CoreHelper.php

```diff
@@ -62,6 +62,9 @@ public static function validateUrl(string $url): string
         $query = substr($url, strlen($protocol));
         $query = preg_replace("#//+#", "/", $query);
 
+        //remove forward slash from end
+        $query = rtrim($query, '/');
```

Comment:
Add the following unit tests for this new behavior:
- pass in a string url with 1 ending forward slash and expect it without it
- pass in a string url with more then 1 ending forward slashes